app/src/main/java/com/example/atheneum/activities/BookInfoActivity.java

```diff
@@ -276,6 +284,9 @@ public void editBook(){
 
     }
 
+    /**
+     * Start activity for deleting the book
```

Comment:
This isn't really an activity, you could just say "trigger viewmodel to delete book" or something